Shared/Samples/Style graphics with renderer/StyleGraphicsWithRendererView.swift

```diff
@@ -16,188 +16,195 @@ import SwiftUI
+                points: [
+                    Point(x: -10e5, y: 40e5),
+                    Point(x: 20e5, y: 50e5)
+                ],
+                spatialReference: .webMercator
+            )
+            
+            // Creates a graphics overlay containing a graphic with the line geometry.
+            let overlay = GraphicsOverlay(graphics: [Graphic(geometry: geometry)])
+            // Creates and assigns a simple renderer to the graphics overlay.
+            overlay.renderer = SimpleRenderer(symbol: symbol)
+            return overlay
+        }()
         
-        // Creates the top left arc segment.
-        let leftArcCenter = Point(x: minX + sideLength * 0.25, y: minY + sideLength * 0.75, spatialReference: spatialReference)
-        let leftArc = EllipticArcSegment.circularEllipticArcEllipticArcSegment(
-            centerPoint: leftArcCenter,
-            radius: arcRadius,
-            startAngle: .pi,
-            centralAngle: -.pi,
-            spatialReference: spatialReference
-        )!
+        /// The graphics overlay rendering a square polygon.
+        private let squarePolygonGraphicsOverlay: GraphicsOverlay = {
+            // Creates a simple fill symbol.
+            let symbol = SimpleFillSymbol(color: .yellow)
+            // Creates the geometry for the square polygon.
+            let geometry = Polygon(
+                points: [
+                    Point(x: -20e5, y: 20e5),
+                    Point(x: 20e5, y: 20e5),
+                    Point(x: 20e5, y: -20e5),
+                    Point(x: -20e5, y: -20e5)
+                ],
+                spatialReference: .webMercator
+            )
+            
+            // Creates a graphics overlay containing a graphic with the square geometry.
+            let overlay = GraphicsOverlay(graphics: [Graphic(geometry: geometry)])
+            // Creates and assigns a simple renderer to the graphics overlay.
+            overlay.renderer = SimpleRenderer(symbol: symbol)
+            return overlay
+        }()
         
-        // Creates the top right arc segment.
-        let rightArcCenter = Point(x: minX + sideLength * 0.75, y: minY + sideLength * 0.75, spatialReference: spatialReference)
-        let rightArc = EllipticArcSegment.circularEllipticArcEllipticArcSegment(
-            centerPoint: rightArcCenter,
-            radius: arcRadius,
-            startAngle: .pi,
-            centralAngle: -.pi,
-            spatialReference: spatialReference
-        )!
+        /// The graphics overlay rendering an ellipse.
+        private let ellipseGraphicsOverlay: GraphicsOverlay = {
+            // Creates a simple fill symbol.
+            let symbol = SimpleFillSymbol(color: .purple)
+            // Defines the center point of the ellipse.
+            let center = Point(x: 40e5, y: 25e5, spatialReference: .webMercator)
+            // Creates the parameters for the ellipse.
+            let parameters = GeodesicEllipseParameters<Polygon>(
```

Comment:
This is a change in `swift/pull/2413`. Without specifying `Polygon` type constraint the returned geometry is a `Multipoint`. Will discuss the reason.